frontend/src/container/OrganizationSettings/AuthDomains/index.tsx

```diff
@@ -266,12 +274,14 @@ function AuthDomains(): JSX.Element {
 						setIsSettingsOpen={setIsSettingsOpen}
 					/>
 				</Modal>
-				<Table
-					rowKey={(record: AuthDomain): string => record.name + v4()}
-					dataSource={!SSOFlag ? notEntripriseData : []}
-					columns={columns}
-					tableLayout="fixed"
-				/>
+				<ResizeTableWrapper columns={columns}>
+					<Table
+						rowKey={(record: AuthDomain): string => record.name + v4()}
```

Comment:
on re render this will create a new function or objects

Reply:
it will return a unique rowKey with function.